go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -31,7 +36,40 @@ import (
 func newClient(cfg config.OvnAuthConfig, dbModel model.ClientDBModel, stopCh <-chan struct{}, opts ...client.Option) (client.Client, error) {
 	const connectTimeout time.Duration = types.OVSDBTimeout * 2
 	const inactivityTimeout time.Duration = types.OVSDBTimeout * 18
-	logger := klogr.New()
+
+	var logger logr.Logger
```

Comment:
maybe use an auxiliary function to set up the logger, it's more than doubling the previous size of the method

Reply:
hm... feels a bit overkill to add a function now @jcaamano . I mean, after removing the sanity check as mentioned in 
https://github.com/ovn-org/ovn-kubernetes/pull/3822#discussion_r1286260220 above, we are adding just one new block to setup lumberjack. PTAL and lmk if you still think an aux function is needed. I have no prob doing it if you think it is a good idea.